### ISSUE TITLE
FormBuilderRadioGroup cannot be reseted 

### ISSUE THREAD

**Issue by supermnemonic** [danvick/flutter_form_builder#647]
I have FormBuilder with FormBuilderRadioGroup as follow.
```
final _radiokey = GlobalKey<FormFieldState>();

FormBuilderRadioGroup<String>(
  key: _radiokey,
  name: 'label_radio',
  initialValue: 'kotlin',
  orientation: OptionsOrientation.wrap,
  wrapAlignment: WrapAlignment.spaceAround,
  options: [
    'Dart',
    'Kotlin',
  ]
      .map((lang) => FormBuilderFieldOption(
            value: lang,
            child: Text('$lang'),
          ))
      .toList(growable: false),
)
```

However, when I reseted form, radio seemed doesn't reset.
```
_formKey.currentState.reset();
// or 
_radiokey.currentState.reset()
```

**Comment by andrewzakhartchouk** [comment]
I can confirm this issue. You can test by trying to use the reset button in the example app of this project.
On the other hadn, the Filter Chip is similar and works as intended with the reset.

**Comment by danvick** [comment]
Fixed in v4.1.0
